toolchain/lower/function_context.h

```diff
@@ -18,10 +19,31 @@ namespace Carbon::Lower {
 // `llvm::Function` definition.
 class FunctionContext {
  public:
-  explicit FunctionContext(FileContext& file_context, llvm::Function* function,
-                           SemIR::SpecificId specific_id,
-                           llvm::DISubprogram* di_subprogram,
-                           llvm::raw_ostream* vlog_stream);
+  explicit FunctionContext(
+      FileContext& file_context, llvm::Function* function,
+      SemIR::SpecificId specific_id, llvm::DISubprogram* di_subprogram,
+      llvm::raw_ostream* vlog_stream,
+      FileContext::SpecificFunctionFingerprint* specific_function_state);
+
+  // Describes a function's body fingerprint while creating the function body.
+  // The final fingerprint is stores in the FileContext as a
+  // SpecificFunctionFingerprint.
+  struct LoweringFunctionFingerprint {
+    // Create a two function fingerprints. Both fingerprints include the data
```

Comment:
A couple things:

- Since this isn't specific to either member, perhaps it should be moved up to the struct?
- Would be nice to have this split into a couple paragraphs, it's a bit dense right now. :)

Reply:
Done, but I don't know how to make the paragraphs more obvious. Empty lines, indentation?